README.md

```diff
@@ -1,53 +1,73 @@
-# ScottyLabs Course Tool
+# CMU Courses
 
-The ScottyLabs Course Tool is a web application designed to aid students at Carnegie Mellon University (CMU) to browse courses, including information about course evaluations and schedules. It is actively maintained by [ScottyLabs](https://scottylabs.org).
+CMU Courses (AKA ScottyLabs Course Tool) is a web application designed to aid students at Carnegie Mellon University in
+browsing courses, including information about course evaluations and schedules. It is actively maintained
+by [ScottyLabs](https://scottylabs.org).
+
+This is the second version of the Course Tool. The old version is currently deployed at [cmucourses.com](cmucourses.com)
```

Comment:
old version is deployed at `cmucourses.com`?

Reply:
oops